modules/check_in/app/serializers/check_in/v2/appointment_data_serializer.rb

```diff
@@ -46,7 +46,22 @@ class AppointmentDataSerializer
           demographics.merge!(emergencyContact: emergency_contact)
         end
 
-        { demographics: demographics, appointments: appointments }
+        if Flipper.enabled?(:check_in_experience_demographics_confirmation_enabled)
+          raw_confirmation = object.payload[:patientDemographicsStatus]
+          demographics_confirmation = {
+            demographicsNeedsUpdate: raw_confirmation[:demographicsNeedsUpdate],
+            demographicsConfirmedAt: raw_confirmation[:demographicsConfirmedAt],
+            nextOfKinNeedsUpdate: raw_confirmation[:nextOfKinNeedsUpdate],
+            nextOfKinConfirmedAt: raw_confirmation[:nextOfKinConfirmedAt],
+            emergencyContactNeedsUpdate: raw_confirmation[:emergencyContactNeedsUpdate],
+            emergencyContactConfirmedAt: raw_confirmation[:emergencyContactConfirmedAt]
+          }
+          { demographics: demographics,
+            appointments: appointments,
+            patientDemographicsStatus: demographics_confirmation }
```

Comment:
minor (just a styling suggestion):
```suggestion
          {
            demographics: demographics,
            appointments: appointments,
            patientDemographicsStatus: demographics_confirmation
          }
```

Reply:
Sure Gaurav, addressed in [98323](https://github.com/department-of-veterans-affairs/vets-api/pull/8722/commits/983233865741a6bb9066c616d695d167e4b609e8)